share/src/main/java/io/zensoft/share/service/facebook/FacebookPublisherManagerService.java

```diff
@@ -23,6 +20,7 @@
     private final DefaultVacancyModelService vacancyModelService;
     private final DefaultVacancyResponseModelService vacancyResponseModelService;
     private final PublisherService facebookPublisherService;
+    private final DefaultVacancyRetriever vacancyRetriever;
```

Comment:
recommend you to replace it with something like `private final VacancyRetriever defaultVacancyRetriever`
cause I guess that was the reason, why it's been realized using inteface

Reply:
Ok, thank you for your comment